pkg/pointerdb/pdbclient/client.go

```diff
@@ -24,7 +24,7 @@ var (
 
 // PointerDB creates a grpcClient
 type PointerDB struct {
-	grpcClient pb.PointerDBClient
+	GrpcClient pb.PointerDBClient
```

Comment:
don't make this public, use a specific constructor instead `NewUsing(` or similar.

Reply:
In general avoid making things public "just so you can test/check by poking at the internals".